package.json

```diff
@@ -5,7 +5,7 @@
     "Jeff Smick"
   ],
   "description": "libxml bindings for v8 javascript engine",
-  "version": "0.14.0",
+  "version": "0.14.1",
```

Comment:
Yep, I did it to prioritize download from my private npm...

On 05/11/2015 06:31 PM, Roman Shtylman wrote:
>
> In package.json 
> <https://github.com/polotek/libxmljs/pull/311#discussion_r30055564>:
>
> > @@ -5,7 +5,7 @@
> >      "Jeff Smick"
> >    ],
> >    "description": "libxml bindings for v8 javascript engine",
> > -  "version": "0.14.0",
> > +  "version": "0.14.1",
>
> Tip: don't do this in PRs. It will never be merged because it is up to 
> the maintainer to decide what the next version will be.
>
> —
> Reply to this email directly or view it on GitHub 
> <https://github.com/polotek/libxmljs/pull/311/files#r30055564>.
>